engine/source/materials/mat_share/usermat_solid.F

```diff
@@ -1595,7 +1595,7 @@ SUBROUTINE USERMAT_SOLID(
           SIGV(I,6) = SVIS(I,6)
         ENDDO
       ENDIF 
-      IF (IVISC > 0) THEN
+      IF (MAT_ELEM%MAT_PARAM(MAT(1))%IVISC >  0) THEN
```

Comment:
@MaciekWronski , do you agree with this change?

Reply:
hello @laurent-altr 
yes, it's an obvious correction, thanks !